Glyssen/Project.cs

```diff
@@ -339,10 +348,58 @@ public Project UpdateProjectFromBundleData(GlyssenBundle bundle)
 				CreateBackup("Backup before updating from new bundle");
 
 			bundle.Metadata.CopyGlyssenModifiableSettings(m_metadata);
+			CopyQuoteMarksIfAppropriate(bundle.WritingSystemDefinition, bundle.Metadata);
 
 			return new Project(bundle, m_recordingProjectName);
 		}
 
+		// internal for testing
+		internal void CopyQuoteMarksIfAppropriate(WritingSystemDefinition targetWs, GlyssenDblTextMetadata targetMetadata)
+		{
+			if (targetWs == null)
+				targetWs = new WritingSystemDefinition();
+
+			// If the target has no quote information, add it.
+			if (!targetWs.QuotationMarks.Any())
+			{
+				targetWs.QuotationMarks.AddRange(WritingSystem.QuotationMarks);
+			}
+			// Assumes QuoteSystemStatus has already been set from source metadata.
+			// If the user hasn't changed the QuoteSystemStatus, keep what is in the target.
+			else if ((targetMetadata.ProjectStatus.QuoteSystemStatus & QuoteSystemStatus.Obtained) > 0)
+			{
+			}
+			// Copy if source has more detail
+			else
+			{
+				bool copy = false;
+				var sourceLevelsCount = WritingSystem.QuotationMarks.Count;
+				var targetLevelsCount = targetWs.QuotationMarks.Count;
+
+				if (sourceLevelsCount >= targetLevelsCount)
```

Comment:
I think this should be strictly greater than. (Though there should be no harm in copying it if it's identical, so it probably doesn't matter in practice.)